mappings/net/minecraft/server/WorldStem.mapping

```diff
@@ -2,7 +2,9 @@ CLASS net/minecraft/unmapped/C_tljyhzqp net/minecraft/server/WorldStem
 	FIELD f_hmwuaxvg saveProperties Lnet/minecraft/unmapped/C_jkevluio;
 	FIELD f_sphgqqzg resourceManager Lnet/minecraft/unmapped/C_carvbxzj;
 	FIELD f_twvealqc resources Lnet/minecraft/unmapped/C_migzkpst;
+	FIELD f_yskbtpkd registries Lnet/minecraft/unmapped/C_bcpxdrik;
 	METHOD close close ()V
 	METHOD m_pfrxpzhv resources ()Lnet/minecraft/unmapped/C_migzkpst;
+	METHOD m_qvpzrpcg registries ()Lnet/minecraft/unmapped/C_bcpxdrik;
```

Comment:
```suggestion
	METHOD m_qvpzrpcg getRegistries ()Lnet/minecraft/unmapped/C_bcpxdrik;
```
[CONVENTIONS.md "getters"](https://github.com/QuiltMC/quilt-mappings/blob/1.20-pre6/CONVENTIONS.md#getters-setters-withers-and-creators)
> Use "get" for non-boolean getters and other methods that calculate some property with no side effects other than caching a value in a private field.

Reply:
this convention doesn't apply to records, since it's superceded by the convention of the java names